src/Thanks.php

```diff
@@ -56,6 +56,7 @@ public function activate(Composer $composer, IOInterface $io)
             }
 
             $app->add(new Command\ThanksCommand());
+            $app->add(new Command\FundCommand());
```

Comment:
@nicolas-grekas could we detect an existing fund command to avoid overriding the composer one once it has support built-in ?

Reply:
I don't know, we can see when it's released :)

Reply:
Looks like doing it only `if (!$app->has('fund'))` would be safer.

Reply:
I thought it would be more complex :)
See #76

Reply:
Released as v1.2.3